src/core/execution/QuickChatExecution.ts

```diff
@@ -10,7 +10,7 @@ export class QuickChatExecution implements Execution {
     private sender: Player,
     private recipientID: PlayerID,
     private quickChatKey: string,
-    private variables: Record<string, string>,
+    private variables: string | undefined,
```

Comment:
I think this should be:
```suggestion
    private target: PlayerID | undefined,
```